packages/flutter_markdown/test/blockquote_test.dart

```diff
@@ -73,23 +73,29 @@ void defineTests() {
         expect(styledTextParts[0].text, 'this is a link: ');
         expect(
           styledTextParts[0].style!.color,
-          theme.textTheme.bodyMedium!.color,
+          styleSheet.p!.color,
```

Comment:
The tests in question are intended to assert that the stylesheet's values are reflected in the widget's output. It made sense to assert the stylesheet over the textTheme, as I thought this would make the test more robust if the text theme is changed by the stylesheet is not.